docs/contracts/functions.rst

```diff
@@ -11,8 +11,8 @@ Functions
 Function Parameters and Return Variables
 ========================================
 
-As in JavaScript, functions may take parameters as input. Unlike in JavaScript
-and C, functions may also return an arbitrary number of values as output.
+Functions take typed parameters as input and may, unlike in many other
```

Comment:
Is "parameters with types" potentially slightly clearer?

Reply:
I think I still prefer the current way it is written.